include/SFML/Audio/SoundBuffer.hpp

```diff
@@ -23,17 +23,19 @@
 ////////////////////////////////////////////////////////////
 
 #ifndef SFML_SOUNDBUFFER_HPP
-#define SFML_SOUNDBUFFER_HPP
+    #define SFML_SOUNDBUFFER_HPP
 
-////////////////////////////////////////////////////////////
-// Headers
-////////////////////////////////////////////////////////////
-#include <SFML/Audio/Export.hpp>
-#include <SFML/Audio/AlResource.hpp>
-#include <SFML/System/Time.hpp>
-#include <string>
-#include <vector>
-#include <set>
+    ////////////////////////////////////////////////////////////
+    // Headers
+    ////////////////////////////////////////////////////////////
+    #include <SFML/Audio/Export.hpp>
+
+    #include <SFML/Audio/AlResource.hpp>
+    #include <SFML/System/Time.hpp>
+
+    #include <set>
+    #include <string>
+    #include <vector>
```

Comment:
Yeah this is very jarring, I have not seen include directives indented ever before. Would strongly suggest to revert